build/tools/ortelius.md

```diff
@@ -202,6 +202,47 @@ curl "http://127.0.0.1:8080/v2/aggregates?startTime=2020-09-21T00:00:00Z&endTime
 }
 ```
 
+#### TxFee Aggregate
+
+AVAX Aggregate txfee
+
+**Example Call**
+
+```text
+curl "http://127.0.0.1:8080/v2/txfeeAggregates?startTime=2020-09-21T00:00:00Z&endTime=2020-10-21T00:00:00Z"
```

Comment:
Please use `localhost` instead of `127.0.0.1` to be inline w/ the rest of the docs.